pkgs/applications/misc/snowsql/darwin.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
```

Given that this package isn't built, but rather installs from a pre-built binary  use of `stdenvNoCC` is likely preferable.
```suggestion
  stdenvNoCC,
```